_posts/en/newsletters/2019-06-19-newsletter.md

```diff
@@ -0,0 +1,294 @@
+  system uses simpler altruist watchtowers that don't receive any reward
+  but otherwise provide both the client and server components for
+  complete enforcement.  You can setup a watchtower for your own
+  channels or you can use the watchtowers of reliable friends.  All the
+  configuration parameters necessary are documented in LND's runtime
+  help.  Developer Will O'Beirne also has an [example tutorial][watchtower
+  tutorial] that helps you setup watchtowers and then observe them
+  protect your money when you attempt a channel breach (his tutorial
+  refers to the PR's code branch but you can now use it with the LND
+  master development branch).
+
+- [LND #3140][] adds support for RBF and CPFP fee bumping when LND is
+  sending transactions using one or more of its onchain UTXOs.
+
+- [LND #3134][] allows users to integrate LND with the [Prometheus][]
+  monitoring solution for collecting statistics and sending alerts.
+
+- [C-Lightning #2672][] adds new RPCs that can be used to open a channel
+  using funds from an external wallet.  The `fundchannel_start` RPC
+  starts opening a new channel with a specified node and returns a
+  bech32 address that the external wallet should pay *using only segwit
+  inputs and without broadcasting the transaction.*  When that
+  transaction has been created, it's serialized form can be provided to
+  the `fundchannel_complete` RPC to securely finish the channel
+  negotiation and broadcast the transaction.  Alternatively, the
+  `fundchannel_cancel` RPC can be called to abort the channel setup
+  before funds are sent.  Because most external wallets will broadcast
+  transactions automatically, these options need to be enabled
+  explicitly using a configuration option---but they make it possible
+  for external wallets to better integrate with C-Lightning directly.
+
+- [C-Lightning #2700][] limits the amount of gossip requested by only
+  requesting gossip from a subset of a node's peers.  This continues the
+  work across all major LN implementations of trying to reduce the
+  amount of data sent via gossip now that the network has grown to
+  thousands of peers.
+
+- [C-Lightning #2699][] extends the `fundchannel` RPC with a new `utxo`
+  parameter that allows the user to specify which of the UTXOs
+  from C-Lightning's built-in wallet to use to fund a new channel.
+
+- [C-Lightning #2696][] adds a new `listtransactions` RPC method that
+  lists all the onchain transactions created by the program and what
+  they were used for (setup, unilateral close, mutual close, or
+  anti-cheat).  Other changes in this PR ensure that all the necessary
+  data to provide these results is stored in the database.
+
+- [Eclair #1009][] allows Eclair to search through gossiped node
+  announcements to find the IP address of channel peers by their pubkeys
+  in case any peer gets disconnected and has its IP address change.
+
+- [BIPs #555][] adds [BIP136][] for bech32-encoded transaction positions
```

Comment:
I haven't looked into this BIP or mailing list post. Do you think it's generally interesting for our readers? I haven't heard anyone talking about it before.

Reply:
A redundant question---I only put things in the newsletter if I think they're generally interesting for the readers.  Of course, I sometimes have strange tastes, so it's good to question them.  :-)  In this case I'm adding an extra sentence describing a possible application with a link to details so that the motivation is clearer.

For discussion, unfortunately most of that seems to have occurred on the PR rather than the mailing list where it would've been more appropriate; still, if you look at the PR you'll find it was well discussed---102 comments from many well-known contributors.

Reply:
Thanks! I should have phrased this as "I'm not convinced that this is generally interesting. Can you give some justification please?", which you've done. Looks good to me now, except with the observation that you've repeated 'proposed' in the sentence. Perhaps reword as:

"This idea was first proposed to the Bitcoin-Dev mailing list over two years ago. A suggested use-case is identifying which transactions contain information useful to third-party applications, such as timestamped decentralized identity references." or similar.